lib/index.js

```diff
@@ -63,7 +63,16 @@ MapboxGeocoder.prototype = {
     minLength: 2,
     reverseGeocode: false,
     limit: 5,
-    origin: 'https://api.mapbox.com'
+    origin: 'https://api.mapbox.com',
+    render: function(feature){
+      var maki;
+      if (feature.properties.maki) maki = feature.properties.maki;
+      else maki = 'marker'
+      return `<img src="https://unpkg.com/@mapbox/maki@6.1.0/icons/${maki}-11.svg"/> ${feature.place_name}`;
+    },
+    getItemValue: function(item) { 
+      return item.place_name
+    }
```

Comment:
Is this just for your testing, or are you intending to default to showing an icon?

Reply:
Still just a WIP -- I'm planning on changing this back to match the existing functionality before merge. 

It might be helpful to use this as an example [here](https://docs.mapbox.com/mapbox-gl-js/examples/#geocoder) since it would demonstrate the custom render function and be a use case I suspect people are pretty familiar with.